src/main/java/sirius/biz/importer/txn/ImportTransactionData.java

```diff
@@ -43,7 +41,6 @@ public class ImportTransactionData extends Composite {
      */
     public static final Mapping SOURCE = Mapping.named("source");
     @NoJournal
-    @AutoImport(hidden = true)
```

Comment:
Wouldn´t they appear now in the job-documentations?

Reply:
No, that has no impact on that. See `sirius.biz.protocol.JournalData#fetchJournaledAndChangedProperties`